backend/routes/content.py

```
@@ -283,6 +283,7 @@ def mark_module_complete_and_open_next(module_id, user_id):
 
         # Check if the module is already marked as complete
         if user_module.completed:
+            logger.debug(f"Module {module_id} is already marked as complete")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)